e2e/app/google-login.ts

```diff
@@ -91,11 +91,13 @@ export default class GoogleLoginPage extends BasePage {
    */
   async load(): Promise<void> {
     const url = configs.uiBaseUrl + configs.loginUrlPath;
-    await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 30000}).catch((err) => {
+    try {
+      await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 30000});
```

Comment:
nit: Maybe wrap this into a higher-level function like `this.navigate(url)`.

The timeout seems a little generous. to my untrained eye, especially for the login page.